libraries/ui-library/src/components/six-avatar/six-avatar.scss

```diff
@@ -28,7 +28,7 @@
   outline: none;
 
   &:hover {
-    box-shadow: var(--six-elevation-4dp);
+    box-shadow: var(--six-elevation-2dp);
```

Comment:
How is this change related to six-select? 🤔

Reply:
I tweaked the base shadow CSS variables slightly to achieve a cleaner appearance for the dropdown. This change also affected how other components looked, so I adjusted their shadows too for consistency. 

After looking at it again, I noticed we have 16 (!) different shadow styles in the base CSS, which is way too many. Most aren't even used, so I now reduced the number of shadow types to three (small, medium and large). See commit 0bb01908d7b0485a0e6ec5f03042eeef41cd2462.